libs/net/entities/Source.cs

```diff
@@ -45,6 +45,17 @@ public class Source : BaseType<int>
     /// </summary>
     public virtual User? Owner { get; set; }
 
+    /// <summary>
+    /// get/set - Foreign key to product.
+    /// </summary>
+    [Column("product_id")]
+    public int? ProductId { get; set; }
+
+    /// <summary>
+    /// get/set - The product associated with this data source..
+    /// </summary>
+    public Product? Category { get; set; }
```

Comment:
It's not a `Category`

Reply:
I realized this wasn't an ideal name, but I couldn't call it Product as the class is Product. What name do you suggest?